boards/remote-pa/Makefile.include

```diff
@@ -0,0 +1,7 @@
+USEMODULE += remote-common
+
+# define the default port depending on the host OS
+PORT_LINUX  ?= /dev/ttyUSB1
```

Comment:
Nop, the prototype A uses a dual channel FTDI chip and the UART0 backchannel uses `/dev/ttyUSB1` typically